.github/ISSUE_TEMPLATE/pvc_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "User support: JupyterHub HW requirements"
+about: "Template for users of JupyterHub requesting HW quota upgrade: disk size, CPU/GPU/Memory."
+title: ""
+labels: user-support
+assignees: ""
+---
+
+### Questionnaire
+
+1. **User details**:
+   Your MOC user name and which applications deployed by our team you're planning to use the bucket at
```

Comment:
This doesn't match the template

Reply:
Isn't the application here "jupyterhub"?

Reply:
That's because I'm bad at copy pasting.. :smile: